src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -230,7 +230,7 @@ public function setCircularReferenceHandler(callable $circularReferenceHandler)
      *
      * @throws InvalidArgumentException if a non-callable callback is set
      */
-    public function setCallbacks(array $callbacks)
+    public function setCallbacks($callbacks)
```

Comment:
This is a BC break. And I'm not sure it is worth it to change anyway because this method is deprecated.

Reply:
Tests fall without this change because of the new case "Callbacks as ArrayObject".

Should I rewrite tests of this method to exclude this case from it?
Or is it better to do the type checking in the body of the method (`is_array($callbacks) || $callbacks instanceof \ArrayObject`) to prevent any illegal usage?